//! Exception Handling:- try and catch Block

//* try:
// This block contains the code in which we put the code that might contains exceptions.

//* catch:
// Whenever some exception is found in try block, it is immediately thrown to catch() block.

// Now catch() block will have the responsibility to handle the exception.
